source/ed/build_helper.cpp

```diff
@@ -244,8 +244,6 @@ nt::VehicleJourney* VJ::make() {
 VJ& VJ::st_shape(const navitia::type::LineString& shape) {
     assert(shape.size() >= 2);
     assert(stop_times.size() >= 2);
-    assert(stop_times.back().st.stop_point->coord == shape.back());
```

Comment:
Not sure (at all): can't you keep that test, or report it for only some cases?

Reply:
Ok, done in shape tests

Reply:
Yes, this assert can't be true anymore because of the projection of stop points on shape.